core/src/main/kotlin/org/evomaster/core/problem/rest/OpenApiAccess.kt

```diff
@@ -62,17 +63,40 @@ object OpenApiAccess {
     }
 
     private fun readFromDisk(openApiUrl: String) : String {
+
+        // file scheme
         val fileScheme = "file:"
-        val path = if (openApiUrl.startsWith(fileScheme, true)) {
-            Paths.get(URI.create(openApiUrl))
-        } else {
-            Paths.get(openApiUrl)
+
+        // path variable
+        var path: Path? = null;
+
+        // checking the given URL
+        try {
+            path = when (openApiUrl.startsWith(fileScheme, true)) {
```

Comment:
this looks bit weird... why replacing an `if` statement with a `when` on booleans?

if it was due to the try/catch, that can be written as:

```
val path = try { if... 
                 catch(...)
```

Reply:
Hi Andrea,

Thank you so much for your comment. I have rewritten the variable declaration according to the way you mentioned.